frontend/app/router.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+import config from './config/environment';
+
+const Router = Ember.Router.extend({
+  location: config.locationType,
+  rootURL: config.rootURL
+});
+
+Router.map(function() {
+  this.route('signIn');
```

Comment:
Is there a rename tool, or should I just manually rename everything? googling hasn't given me much to go on.